django_oikotie/enums.py

```diff
@@ -12,32 +12,37 @@ class ApartmentAction(Enum):
     REMOVE = "remove"
 
 
+class Availability(Enum):
+    pre_marketing = "ennakkomarkkinoinnissa"
```

Comment:
Are these left lower case intentionally?.

Reply:
Ah, no they're not. I'll fix those.

Reply:
And fixed. Found other enums with lowercase names as well.